docs/release-checklist.rst

```diff
@@ -153,13 +158,18 @@ need to be uploaded to https://tahoe-lafs.org in `~source/downloads`
 - secure-copy all release artifacts to the download area on the
   tahoe-lafs.org host machine. `~source/downloads` on there maps to
   https://tahoe-lafs.org/downloads/ on the Web.
-- scp dist/*1.15.0* username@tahoe-lafs.org:/home/source/downloads
+    - scp dist/*1.15.0* username@tahoe-lafs.org:/home/source/downloads
+
 - the following developers have access to do this:
 
   - exarkun
   - meejah
   - warner
 
+Push the signed tag to the main repository:
+
+- git push origin_push tahoe-lafs-1.17.1
```

Comment:
`origin_push` is an unusual name for that remote isn't it?  Should the doc stick with this name or use the conventional `origin`?

Reply:
Mmm, yeah I guess .. 99% of the commands in here are for future-me to copy-paste though ;) but I guess that's an unusual name (which I use so I don't accidentally push to "origin" when I meant "github" which is what I always call my fork of a thing)